libs/designer/src/lib/core/actions/bjsworkflow/serializer.ts

```diff
@@ -303,9 +308,9 @@ const getOperationInputsToSerialize = (rootState: RootState, operationId: string
   }));
 };
 
-const serializeOperationParameters = (inputs: SerializedParameter[], manifest: OperationManifest): Record<string, any> => {
+const serializeParametersFromManifest = (inputs: SerializedParameter[], manifest: OperationManifest): Record<string, any> => {
   const inputsLocation = (manifest.properties.inputsLocation ?? ['inputs']).slice(1);
-  const inputPathValue = constructInputValues('inputs.$', inputs, false /* encodePathComponents */);
+  const inputPathValue = constructInputValues('inputs.$', inputs, { encodePathComponents: false, flattenPaths: true });
```

Comment:
Please do not do it like this, you are assuming all manifest code path should follow this, this is only specific to parameters which have the parameterAlias. Use that property "alias" from serializedParameter in the code logic method "serializeParameter" to determine if you need to flatten it or not. This would fail all existing manifests.

Reply:
@preetriti1 There is no `alias` property on the parameter. Are you referring to `info.alias`?

![image](https://user-images.githubusercontent.com/1350074/229858083-20b31d32-067e-4fd0-b9ad-d4fd25d95557.png)

Reply:
Yes info.alias,

Swagger or manifest can have any property name so I can name a property to have slashes, but it should not be assumed to be open api... please move your logic for aliasing specific, we should not assume and do pattern matching in client code unless there is no other way, this has lead to bugs before.

Reply:
Fixed. Need to re-run my E2E scenarios but tests are passing.